tools/c7n_azure/c7n_azure/actions.py

```diff
@@ -78,22 +97,46 @@ def process(self, resources):
             for key in new_tags:
                 tags[key] = new_tags[key]
 
-            # resource group type
-            if self.manager.type == 'resourcegroup':
-                params_patch = ResourceGroupPatchable(
-                    tags=tags
-                )
-                client.resource_groups.update(
-                    resource['name'],
-                    params_patch,
-                )
-            # other Azure resources
-            else:
-                az_resource = GenericResource.deserialize(resource)
-                api_version = session.resource_api_version(az_resource.id)
-                az_resource.tags = tags
+            update_resource_tags(self, session, client, resource, tags)
+
+
+class RemoveTag(BaseAction):
+    """Removes tags from Azure resources
+
+        .. code-block:: yaml
+
+          policies:
+            - name: azure-remove-tag-resourcegroups
+              resource: azure.resourcegroup
+              description: |
+                Remove tag for all existing resource groups with a key such as Environment
+              actions:
+               - type: tag
+                 tag: Environment
+    """
+    schema = utils.type_schema(
+        'untag', aliases=('unmark', 'remove-tag'),
```

Comment:
I confirmed with @kapilt that we didn't need these aliases. (It was for backward compatibility)

Reply:
Oh cool, will delete!